packages/scandipwa/src/store/Cart/Cart.action.js

```diff
@@ -80,3 +81,9 @@ export const applyCouponToCart = (couponCode) => ({
 export const removeCouponFromCart = () => ({
     type: REMOVE_COUPON_FROM_CART
 });
+
+/** @namespace Store/Cart/Action/updateIsLoading */
+export const updateIsLoading = (isLoading) => ({
+    type: UPDATE_TOTALS_IS_LOADING,
+    isLoading
+});
```

Comment:
make semantically correct name of function, like updateIsLoadingCart, same for type